README.md

```diff
@@ -1,2 +1,78 @@
-# Hello-Project
-My first GitHub repo!
+<a name="readme-top"></a>
+<div align="center">
+  <h3><b>Hello-Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Project] <a name="about-project"></a>
+
+**[Hello-Project]** is to analyze source code and provide feedback on potential errors, coding style violations, other issues and correct them.
+
+### Key Features <a name="Error Detection"></a>
+
+ Analyze the source code and identify potential errors, such as syntax errors, undefined variables, unused variables, and type mismatches. By detecting these errors early, linters help developers catch and fix bugs before they cause runtime issues.
+
+- **[Code complexity analysis]**
+- **[Error detection]**
+- **[Code formatting and auto-fixing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"> will be added soon.</a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="Muhammad Usman Tariq"></a>
+
+👤 **Muhammad Usman Tariq**
+
+- GitHub: [@githubhandle](https://github.com/Muhammad-UsmanTariq/Hello-Project/pull/1)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/osman-tariq-8a7543266/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ optional] Next time use your usename in the [] brackets in Authors' section @Muhammad-UsmanTariq 🍔